File: MiniDOMWebCtrls/DMD/Luoghi/CComuniModuleHandler.cs

namespace minidom.Forms
{
    public class CComuniModuleHandler : CBaseModuleHandler
    {
        public CComuniModuleHandler() : base(ModuleSupportFlags.SCreate | ModuleSupportFlags.SDelete | ModuleSupportFlags.SDuplicate | ModuleSupportFlags.SEdit | ModuleSupportFlags.SAnnotations)
        {
        }

        public override Databases.DBObjectCursorBase CreateCursor()
        {
            var ret = new Anagrafica.CComuniCursor();
            ret.Nome.SortOrder = Databases.SortEnum.SORT_ASC;
            return ret;
        }
    }
}

File: MiniDOMWebCtrls/DMD/Luoghi/CComuniModuleHandler.vb
Imports minidom
Imports minidom.Sistema
Imports minidom.WebSite

Imports minidom.Forms
Imports minidom.Anagrafica
Imports minidom.Databases
Imports minidom.Forms.Utils
Imports minidom.XML

Namespace Forms

    Public Class CComuniModuleHandler
        Inherits CBaseModuleHandler

        Public Sub New()
            MyBase.New(ModuleSupportFlags.SCreate Or ModuleSupportFlags.SDelete Or ModuleSupportFlags.SDuplicate Or ModuleSupportFlags.SEdit Or ModuleSupportFlags.SAnnotations)
        End Sub

        Public Overrides Function CreateCursor() As DBObjectCursorBase
            Dim ret As New CComuniCursor
            ret.Nome.SortOrder = SortEnum.SORT_ASC
            Return ret
        End Function


    End Class


End Namespace